src/XamlStyler.Console/XamlStyler.Console.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <PackageId>XamlStyler.Console</PackageId>
-    <PackageVersion>3.1.0.0</PackageVersion>
+    <PackageVersion>3.1.1.0</PackageVersion>
```

Comment:
Nit: Can you revert this change? Our build pipeline will overwrite versioning for release.